src/pymedphys/dicom/_level2/anonymise.py

```diff
@@ -213,6 +215,61 @@ def anonymise_dicom_dataset(ds, replace_values=True, delete_private_tags=True,
     return ds_anon
 
 
+def anonymise_dicom_file(dicom_filepath, overwrite_file=False, replace_values=True,
+                         keywords_to_leave_unchanged=None, delete_private_tags=True,
+                         delete_unknown_tags=None):
+
+    ds = pydicom.dcmread(dicom_filepath)
+
+    if overwrite_file:
+        dicom_anon_filepath = dicom_filepath
```

Comment:
You mentioned you always wanted to append annonymised at the end? I've come around. Happy for that to be the case, so it seems the logic should be to pass a flag "delete original files" or something of the sort. And the steps would be to always append anonymised to the file name, but delete the old if the flag asks for it.

Reply:
I was actually only suggesting that we append anonymised for non-in-place anonymisation. But now I think on it, maybe it is best to do for all cases and just delete the original files...

Reply:
> I was actually only suggesting that we append anonymised for non-in-place anonymisation. But now I think on it, maybe it is best to do for all cases and just delete the original files...

Sounds good to me :)